src/EditorFeatures/CSharpTest/UseThrowExpression/UseThrowExpressionTests.cs

```diff
@@ -454,5 +454,74 @@ public B(A a)
     }
 }");
         }
+
+        [WorkItem(22926, "https://github.com/dotnet/roslyn/issues/22926")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseThrowExpression)]
+        public async Task TestNotWhenUnconstrainedTypeParameter()
+        {
+            await TestMissingInRegularAndScriptAsync(
+@"using System;
+class A<T>
+{
+    T x;
+    public A(T t)
+    {
+        if (t == null) [|throw|] new ArgumentNullException();
```

Comment:
It seems so wacky that you can write this, but not ```t ?? throw ...```.  It feels like a csharplang bug/suggestion should be filed on this.

Reply:
@CyrusNajmabadi I agree with you. Proposal to fix it https://github.com/dotnet/csharplang/issues/263